src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -1,32 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <!-- Perf Runner NuGet package paths -->
+  <!-- Perf Runner NuGet package paths -->
+  <PropertyGroup>
+   <XunitPerfAnalysisPackageVersion Condition="'$(XunitPerfAnalysisPackageVersion)' == ''">1.0.0-alpha-build0037</XunitPerfAnalysisPackageVersion>
```

Comment:
While this is not necessarily tied to your change, I'm curious how this stays in-sync with individual repo's? I really dislike having this version number hard-coded in buildtools targets files because it is fragile. Should we instead expect that is be set and if not then given an error?